_locales/_untranslated_en.json

```diff
@@ -1 +1,5 @@
-{}
+{
+  "download_failed": {
+    "message": "Download failed"
```

Comment:
There's no translation for this?

Reply:
not yet, that is missing somehow, and the state seems to be hard to achieve anyway, maybe there is a core bug, not sure.

however, as the missing string produces some noise, i just added it :) https://github.com/deltachat/deltachat-android/pull/2167 - however, i do not think it is super-important to pull it in for desktop, it would probably hardly be shown anyway,

Reply:
nice, we can then just remove it from `_untranslated_en.json` when we pull the translation strings again as it has the same key.